### PR TITLE
Issue #40: added feature to purge: empty trash now

### PR DESCRIPTION
**Pull request by parthsuresh**

### CODE REVIEW

File: src/functions/purging_rmw.c [hunk 1]
```diff
@@ -252,6 +252,11 @@ purge (const short purge_after, const struct waste_containers *waste, char *time
 
   short status = 0;
 
+  bool cmd_empty = 0;
+
+  if (getenv("RMWTRASH") != NULL)
```
Comment (andy5995) [inline src/functions/purging_rmw.c]: We must check to see if it's not NULL before trying to use it, otherwise we get a segfault.

**Comment by parthsuresh** [comment]
So, what can I do next?

**Comment by andy5995** [comment]
@parthsuresh Next? Well, I realized that if the trash will be emptied, then there isn't any need to read the date from the trashinfo files. In effect, a lot of this code could be skipped over with an if statement.

Only execute the code if (!cmd_empty)...

**Comment by parthsuresh** [comment]
@andy5995 I'm on it..

**Comment by andy5995** [comment]
@parthsuresh Do you have any questions about how to approach this?

**Comment by andy5995** [comment]
I finished this up with https://github.com/andy5995/rmw/commit/511c666de9c7891012755d5e1661129911870189

**Pull request merged**